web/apps/labelstudio/src/pages/Settings/AnnotationSettings.js

```diff
@@ -57,23 +57,13 @@ export const AnnotationSettings = () => {
 
           <Form.Row columnCount={1} style={{ borderTop: '1px solid #f1f1f1' }}>
             <br />
-            <Elem name={'header'}>Live Predictions</Elem>
+            <Elem name={'header'}>Prelabeling</Elem>
             <div>
               <Toggle
-                label="Use predictions to pre-label tasks"
+                label="Use predictions to prelabel tasks"
                 description={
                   <span>
-                    Enable and select which set of predictions to use for prelabeling.
-                    Predictions will be pre-loaded in {" "}
-                    <a
-                      style={{ color: "rgb(105 129 185)" }}
-                      target="_blank" href="https://labelstud.io/guide/labeling.html#Choose-which-tasks-to-label"
-                    >Label&nbsp;All&nbsp;Tasks</a>{" "}
-                    and {" "}
-                    <a
-                      style={{ color: "rgb(105 129 185)" }}
-                      target="_blank" href="https://labelstud.io/guide/get_started#Interface"
-                    >Quick View</a>.
```

Comment:
Looks like a helpful info given the diversity of affected workflows (and I believe some were missing here anyway) + links to documentation.
Can we maybe rephrase this without removal? Why is it better to not have this info at all?

Reply:
The state of those linked docs is not great, and it would create more confusion that explain much. We first need to improve those sections, before we include it into the product. Overall, its simplicity.